### ISSUE TITLE
Add icons for project status

### ISSUE THREAD

**Issue by sarahmonster** [a8cteam51/strikebase#106]
- [ ] Pending: Organisation has been contacted and has expressed interest in project. Project is waiting for either the client to be ready or for us to be ready. (Not sure if these should be split into two? Maybe “ready” for projects that are just waiting for someone to take them on, and “planning” for projects where we’re still speccing out or waiting on client input/readyness?)
- [ ] Cold: Organisation isn’t interested in working with us, or has changed direction.
- [ ] In progress: A project under active development.
- [ ] Launched: New site published.
- [ ] Complete: A project for which work has finished, but no site has launched.
- [ ] On hold/paused

These will be our top-level categories for project status.

**Comment by wrightcj03** [comment]
>  Pending: Organisation has been contacted and has expressed interest in project. Project is waiting for either the client to be ready or for us to be ready. (Not sure if these should be split into two? Maybe “ready” for projects that are just waiting for someone to take them on, and “planning” for projects where we’re still speccing out or waiting on client input/readyness?)

 I like the idea of "ready" to indicate that the project is just waiting to be started. "Pending" could mean that there's still a step outstanding (a proposal, an approval, etc...). So how about:

Pending: Organisation has been contacted and has expressed interest in project. Project is waiting for finalization of project proposal.

Ready: Proposal has been finalized and approved. We can can start on the project. 

> Cold: Organisation isn’t interested in working with us, or has changed direction.

How about: Organisation hasn't responded to our initial inquiry, isn’t interested in working with us, or has changed direction.

**Comment by sarahmonster** [comment]
👍 on these adjustments!

**Comment by michiecat** [comment]
Sounds good. I'm also cool with removing the Launched and Complete statuses, what do you think, @wrightcj03, @sarahmonster? I know I was pushing for that earlier, but after more thinking it seems like additional complexity and potential redundancy. The "Consulting" project type could cover projects that don't involve an actual launch.

**Comment by wrightcj03** [comment]
@michiecat Removing "Complete" sounds good. After all, is anything ever really complete? :joy:

**Comment by sarahmonster** [comment]
Not the way we do it it isn't! ;)

If the combination of "consulting" + "launched" doesn't give us the level of control we need (our filters can show only launched projects of the consulting type) we can always re-evaluate.

**Comment by michiecat** [comment]
These genres have been updated on the site (I filled in the descriptions as well so we have a reminder).

**Comment by sarahmonster** [comment]
Lovely, thank you! :)
